Dockerfile

```diff
@@ -12,4 +12,5 @@ WORKDIR /
 RUN mkdir /config
 
 ONBUILD ADD config.yml /config/
-ENTRYPOINT [ "java", "-jar", "/cloudwatch_exporter.jar", "9106", "/config/config.yml" ]
+ENTRYPOINT [ "java", "-jar", "/cloudwatch_exporter.jar", "9106"]
```

Comment:
Why move the config and not the port?

Reply:
Port mapping can be changed easily with `-p 91xx:9106`

Can move it in to the CMD if you'd like though.

Reply:
Makes sense